cranelift/codegen/src/isa/aarch64/lower_inst.rs

```diff
@@ -654,10 +670,28 @@ pub(crate) fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
                         rn,
                         signed,
                         from_bits,
-                        to_bits,
+                        to_bits: std::cmp::min(64, to_bits),
                     });
                 }
             }
+
```

Comment:
With the above `pass_through_lower`, could we do an `else if pass_through_lower { ctx.emit(/* move */) }` here? I like that arrangement a bit better as it gives a clear separation between "produce lower register" and "produce upper register" parts of the code.